doc/source/conf.py

```diff
@@ -483,6 +484,8 @@ def _autogen_apis(app: sphinx.application.Sphinx):
 
 
 def setup(app):
+    generate_versions_json()
```

Comment:
i think we can limit this function to run only on the readthedocs environment? Couple of ways to detect the readthedocs environment I can think of
- The $READTHEDOCS_OUTPUT is set and not null
- We can add a `pre_build` step to https://github.com/ray-project/ray/blob/master/.readthedocs.yaml (https://docs.readthedocs.io/en/stable/build-customization.html), and run this function as a `pre_build` step instead of inside `conf.py`?